test/src/test/java/jenkins/security/ApiTokenPropertyTest.java

```diff
@@ -50,7 +63,7 @@ public Credentials getCredentials(AuthScheme scheme, String host, int port, bool
             protected HttpClient getHttpClient() {
                 HttpClient c = super.getHttpClient();
                 c.getParams().setAuthenticationPreemptive(true);
-                c.getState().setCredentials(new AuthScope("localhost", localPort, AuthScope.ANY_REALM), new UsernamePasswordCredentials("foo", token));
+                c.getState().setCredentials(new AuthScope("localhost", AuthScope.ANY_PORT, AuthScope.ANY_REALM), new UsernamePasswordCredentials("foo", token));
```

Comment:
Because ``localPort`` is protected in ``JenkinsRule``. So accessing it would have required further changes.

Restricting the ``AuthScope`` that much for testing is not required. Even ``AuthScope.ANY`` would be fine. Anything else I would only use for security reasons on production not to provide credentials to an untrusted server by mistake.